### DISCUSSION TITLE
如何查看archery的所有执行记录，不是审核工单记录。

### DISCUSSION THREAD

**Discussion by zoe-ctrl** [hhyo/Archery#2727] [Q&A]
目前由另一个审核平台也调用了archery的执行功能，需求是想在archery里看到所有的执行记录，包括其他平台调用的，但是目前只能看到archery自己的工单记录（workflow_log）

**Comment by nick2wang** [comment]
你应该是跳过archery直接调用goinception来执行了，这个是没有记录的

**Reply by zoe-ctrl** [reply]
那请问如果不考虑第二个审核平台，只是想看archery的所有执行记录，是不是也只能通过workflow_log来看？好像么有找到execution_log之类的东西

**Reply by nick2wang** [reply]
> 那请问如果不考虑第二个审核平台，只是想看archery的所有执行记录，是不是也只能通过workflow_log来看？好像么有找到execution_log之类的东西

是的，结合工单和日志表看
